### PR TITLE
Update CORs settings in registrar so Program Manager can hit it on stage

### PR DESCRIPTION
**Pull request by reillz10**
## Description

TODO: Include a detailed description of the changes in the body of the PR

### CODE REVIEW

File: registrar/settings/production.py [hunk 1]
```diff
@@ -61,3 +63,7 @@
 if vars().get(jwk_setting) == 'None':
     vars()[jwk_setting] = None
 del jwk_setting
+
+CORS_ALLOW_HEADERS = corsheaders_default_headers + (
```
Comment (iloveagent57) [inline registrar/settings/production.py]: Let's move this into the base settings file.  Both discovery and edx-platform define it there.

File: registrar/settings/base.py [hunk 2]
```diff
@@ -67,7 +67,7 @@
 )
 
 # Enable CORS
-CORS_ALLOW_CREDENTIALS = True
+CORS_ALLOW_CREDENTIALS = False
```
Comment (iloveagent57) [inline registrar/settings/base.py]: Other services (discovery and edx-platform) have this set to True.  Not sure where is better to copypasta from...

Comment (iloveagent57) [reply registrar/settings/base.py]: Also, the comment above becomes a lie when we set this to False.

Comment (iloveagent57) [reply registrar/settings/base.py]: https://github.com/ottoyiu/django-cors-headers/blob/a259cb18247f3ab4655a166483438772b3d66287/README.rst#cors_allow_credentials . I think we need to allow cookies in the cross-site requests for any JWT auth to work, right?

**Review by iloveagent57 — COMMENTED** [review]
I'd set that back to true and move the headers thing into base.py.  Maybe we wanna double-check with @douglashall though.

**Comment by douglashall** [comment]
> I'd set that back to true and move the headers thing into base.py. Maybe we wanna double-check with @douglashall though.

Sounds good to me!

File: registrar/settings/base.py (hunk 2 above)
Comment (reillz10) [reply registrar/settings/base.py]: Ha! the comment one is a funny miss on my end. I'm going to do all the things you said and Doug agreed with

File: registrar/settings/production.py (hunk 1 above)
Comment (reillz10) [reply registrar/settings/production.py]: Done

**Review by iloveagent57 — APPROVED** [review]

**Pull request merged**